core/controlplane/config/config.go

```diff
@@ -680,6 +694,16 @@ type LocalStreaming struct {
 	interval int    `yaml:"interval"`
 }
 
+type NetworkingDaemonSets struct {
+	Enabled         bool        `yaml:"enabled"`
+	Typha           bool        `yaml:"typha"`
+	CalicoNodeImage model.Image `yaml:"calico-node-image"`
```

Comment:
Super nit but would you mind changing yaml keys to lowerCamelCase for consistency with other existing keys?

Reply:
Sure - no problem